### PR TITLE
npm updates

### PR DESCRIPTION
**Pull request by ggetz**
* Updates outdated npm modules
* Updates markdownlint usage (namely the ignore rules) and is moved to a configuration file

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ggetz!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/npm-update/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: package.json [hunk 1]
```diff
@@ -136,7 +135,7 @@
     "build-docs-watch": "gulp buildDocsWatch",
     "eslint": "eslint \"./**/*.js\" \"./**/*.cjs\" \"./**/*.html\" --cache --quiet",
     "make-zip": "gulp makeZip",
-    "markdownlint": "markdownlint \"*.md\" \"Documentation/**/*.md\" \"packages/**/*.md\" --ignore CHANGES.md --ignore \"./**/LICENSE.md\"",
```
Comment (jjhembd) [inline package.json:139]: I like the .markdownlintignore file for CHANGES and LICENSE, etc.--it is a cleaner approach.
But currently markdownlint is checking files like `Documentation/CustomShaderGuide/README.md`. With this change, those files will be ignored. Was that the intent here?

Comment (ggetz) [reply package.json:139]: My mistake; I thought that was an ignored directory.

**Comment by ggetz** [comment]
Thanks @jjhembd! This should be ready for another look!

**Review by jjhembd — APPROVED** [review]
Looks good, thanks @ggetz!

**Pull request merged**